python/sglang/srt/mem_cache/storage/mooncake_store/mooncake_store.py

```diff
@@ -63,10 +71,16 @@ def from_file() -> "MooncakeStoreConfig":
                 config.get("global_segment_size", DEFAULT_GLOBAL_SEGMENT_SIZE)
             ),
             # Zero copy interface does not need local buffer
-            local_buffer_size=DEFAULT_LOCAL_BUFFER_SIZE,
+            local_buffer_size=config.get(
+                "local_buffer_size", DEFAULT_LOCAL_BUFFER_SIZE
```

Comment:
Why do we need local_buffer_size parameter?

Reply:
Keep this to be compatible with non-zerocopy calls, such as warmup().